params/avalanche_params.go

```diff
@@ -32,6 +31,9 @@ const (
 	ApricotPhase5TargetGas                uint64 = 15_000_000
 	ApricotPhase5BaseFeeChangeDenominator uint64 = 36
 
+	DynamicFeeExtraDataSize        = 80
+	RollupWindow            uint64 = 10
```

Comment:
doesn't seem this is referenced in the code.

Reply:
This is migrated in the next PR